app/components/modal/order/order-cart.html

```diff
@@ -0,0 +1,65 @@
+<div class="modal-header">
+    <button type="button" class="close" ng-click="orderModalCtrl.modalCancel()">&times;</button>
+    <h3 ng-hide="orderModalCtrl.order == '' || orderModalCtrl.order == undefined" class="modal-title" id="modal-title">
+        Confirm Order</h3>
+</div>
+<div class="modal-body" id="modal-body">
+
+    <table class="table table-condensed table-hover table-striped table-bordered ">
+        <caption ng-hide="orderModalCtrl.order == '' || orderModalCtrl.order == undefined">
```

Comment:
use ng-show="orderModalCtrl.order" instead of ng-hide="orderModalCtrl.order == '' || orderModalCtrl.order == undefined", please fix this in other file also